SubsoniciousKit/SubsoniciousKit/Network/Service.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Service.swift
+//  SubsoniciousKit
+//
+//  Created by Bilal on 10/07/2020.
+//  Copyright © 2020 Bilal Benlarbi. All rights reserved.
+//
+
+import Foundation
+
+public class Service {
+
+    let session: URLSession
+
+    public init(session: URLSession = URLSession(configuration: .default)) {
```

Comment:
maybe just the configuration in the init ?

Reply:
why not